web/web-frontend/src/components/project/rest/method/modal/UpdateMethodModal.js

```diff
@@ -84,7 +85,8 @@ class UpdateMethodModal extends PureComponent {
     onForwardedEndpointChange(forwardedEndpoint){
         this.setState({ updateMethod: {
                 ...this.state.updateMethod,
-                forwardedEndpoint: forwardedEndpoint
+                forwardedEndpoint: forwardedEndpoint,
+                automaticForward: forwardedEndpoint ? this.state.updateMethod.automaticForward : false
```

Comment:
This allows us to ensure that if the forwarded endpoint is changed to an empty value then we also disable the automatic forward feature since it wouldn't make sense to forward.